src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskThankYouCollectionViewCell.xib

```diff
@@ -66,9 +85,26 @@
                                         </userDefinedRuntimeAttributes>
                                     </label>
                                     <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="BEI-EV-nwa">
-                                        <rect key="frame" x="0.0" y="296" width="341" height="173"/>
+                                        <rect key="frame" x="16" y="412" width="396" height="57"/>
+                                        <constraints>
+                                            <constraint firstAttribute="width" constant="396" id="0DF-hU-67w"/>
```

Comment:
Please don't set fixed `width` constraints on all these. We support multiple screen sizes and get auto layout warnings because of these. Your solution should work without them

Reply:
I had a shot discussion with Andreas about that and we came to a solution with a proportional width constraint used

Reply:
I tried it out before I commented, there already were trailing and leading constraints, you just would have needed to delete the redundant width constraints, that's all. Would be a cleaner solution than involving the superview's width as it is now.

Reply:
removed the fixed width. No more layout warnings at runtime.

Reply:
Discussed with Nick; removed the width, added constraints for horizontal layout.